.github/workflows/build-container.yaml

```
@@ -0,0 +1,83 @@
+name: Build Deploy Container
+
+on:
+
+  # Always have a base image ready to go - this is a nightly build
+  schedule:
+    - cron: 0 3 * * *
+
+  # Allow manual trigger of a build
+  workflow_dispatch:
+    inputs:
+      whatever:
+        description: "This variable does not matter, it's a GH-actions bug."
```

Looking at https://github.com/spack/spack/blob/999eee64b8d36b303a8b13d9d50d422120da355b/.github/workflows/build-containers.yml#L3-L20, could this just be

```suggestion
  workflow_dispatch:
```

---

It should work like that, but when I've tried that it doesn't work to trigger it, I think it's a bug. That's why I added the variable.

---

I tested it here: https://github.com/tzanio/gha-test/actions

---

Then go for it - it could be my browser or something.